packages/amplify-category-api/resources/awscloudformation/overrides-resource/APIGW/package.json

```diff
@@ -8,11 +8,9 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "dependencies": {
-    "fs-extra": "^9.1.0"
+    "@aws-amplify/cli-extensibility-helper": "2.3.0-apiext3.0"
   },
   "devDependencies": {
-    "@aws-amplify/cli-extensibility-helper": "2.3.0-apiext3.0",
-    "@types/fs-extra": "^9.0.11",
     "typescript": "^4.2.4"
   }
-}
+}
```

Comment:
```suggestion
}

```

Reply:
Ahh this  change is always confusing , Some says to remove blank spaces

Reply:
POSIX standard says all lines in a text file should end in a newline, but it's not a major issue.

Reply:
done